daemon/archive.go

```diff
@@ -257,12 +269,6 @@ func (container *Container) ExtractToDir(path string, noOverwriteDirNonDir bool,
 		return ErrExtractPointNotDirectory
 	}
 
-	baseRel, err := filepath.Rel(container.basefs, resolvedPath)
-	if err != nil {
-		return err
-	}
-	absPath = filepath.Join(string(os.PathSeparator), baseRel)
-
```

Comment:
ooh, this is actually a bit tricky to test using `docker cp` so I'll have to test it using the API directly.